src/functionalTest/resources/features/F-204 - Get NoC Questions/common/204_NoC_WithoutEvents_Case_Creation_Token_Creation.td.json

```diff
@@ -0,0 +1,29 @@
+{
+	"_guid_": "204_NoC_WithoutEvents_Case_Creation_With_Org_Policies_Token_Creation",
+  "_extends_" : "Case_Creation_Token_Creation",
```

Comment:
NB: you could have extended: `NoC_WithoutEvents_Case_Creation_Token_Creation__Base` to make this call simpler.